client/dist/index.html

```diff
@@ -1 +1,11 @@
-<h1>TEST</h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+<header>
+
+</header>
+<body>
+  <h1>HELLOOO</h1>
+  <div id="photos"></div>
+  <script src='bundle.js'></script>
```

Comment:
Aren't double-quotes the standard for .html files, especially since you aren't explicitly writing any javascript here?

Reply:
Changed!